docker_posix.yml

```diff
@@ -219,6 +219,55 @@ properties:
           title: Exit codes
           type: integer
           minimum: 1
+  image:
+    title: Docker image.
+    description: Image to use for the task.  Images can be specified as an image tag
+      as used by a docker registry, or as an object declaring type and name/namespace
+    oneOf:
+    - title: Docker image name
+      type: string
+    - type: object
+      title: Named docker image
+      properties:
+        type:
+          type: string
+          enum:
+          - docker-image
+        name:
+          type: string
+      required:
```

Comment:
At the same level as `required:` you will also need `additionalProperties: false`, to disallow that other properties are included.